packages/server/src/api/controllers/row/alias.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  QueryJson,
+  SearchFilters,
+  Table,
+  Row,
+  DatasourcePlusQueryResponse,
+} from "@budibase/types"
+import { getDatasourceAndQuery } from "../../../sdk/app/rows/utils"
+import { cloneDeep } from "lodash"
+
+class CharSequence {
```

Comment:
This seems pretty critical. Do we have tests for it, specially around what happen when it goes over all the alphabet characters?

Reply:
Yeap - there is a test in the sqlAlias.spec.ts for this!